src/main/webapp/github-scm-source.js

```diff
@@ -0,0 +1,49 @@
+// We match the end of the name using $= because the beginning of the name is 
+// dynamically generated to avoid name clashes.
+Behaviour.specify("input[name$=_isConfiguredByUrl]", 'GitHubSCMSourceRadioConfiguration', 0, function(e) {
+    if (e.gitHubSCMSourceRadioConfiguration) {
+        return;
+    }
+
+    var getNthParent = function(e, n) {
+        while (n > 0) {
+            if (e.parentNode) {
+                e = e.parentNode;
+                n--;
+            } else {
+                return null;
+            }
+        }
+        return e;
+    }
+
+    // Todo: Replace with a query selector?
+    var findNeighboringDynamicInput = function(e) {
+
+        var inputTbody = getNthParent(e, 4 /*tbody > tr > td > label > input*/);
+        if (inputTbody) {
+            var radioblockStart = getNthParent(e, 3 /* tr > td > label > input*/);
+            // input hidden is always in the 4th position
+            var hiddenBlock = radioblockStart.parentNode.childNodes[4].firstElementChild.firstElementChild
```

Comment:
Isn't `radioblockStart.parentNode` the same as `inputTBody`? Seems unnecessary to compute both values.